XTS/wsat-jta-multi_hop/pom.xml

```diff
@@ -219,22 +220,22 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
                             <skip>false</skip>
+                            <argLine>${ipv6.server.jvm.args}</argLine>
```

Comment:
Why do you need to set the args on the surefire VM and on the launched arquillian server